cmd/up/up.go

```diff
@@ -502,7 +502,10 @@ func (up *upContext) waitUntilExitOrInterrupt() error {
 				if errors.IsTransient(err) {
 					return err
 				}
-				return errors.ErrCommandFailed
+				return errors.CommandError{
+					E:        errors.ErrCommandFailed,
+					ExitCode: err,
```

Comment:
Since this isn't the exit code but a string, maybe rename it to "Reason" or "OriginalErr"?